src/app/utils/GDPRUserList.js

```diff
@@ -0,0 +1,6 @@
+const list = `
```

Comment:
Would be nice to add a comment to indicate that this expects a list separated by new-lines... at least until some names are added?

Reply:
this is a convention used in all other user lists in this codebase